proto/account/command.proto

```diff
@@ -68,3 +69,15 @@ message ChangeAPIKeyNameCommand {
 message EnableAPIKeyCommand {}
 
 message DisableAPIKeyCommand {}
+
+message CreateSearchFilterCommand {
+  string name = 1;
+  string query = 2;
+  account.FilterTargetType filter_target_type = 3;
+  string environment_id = 4;
+  bool default_filter = 5;
+}
+
+message UpdateSearchFilterCommand {
+  SearchFilter search_filter = 1;
```

Comment:
As discussed, we will only pass the fields which will be updated like in the `CreateSearchFilterCommand`.